admin/jsonadm/src/Admin/JsonAdm/Common/Factory/Base.php

```diff
@@ -72,9 +72,9 @@ protected static function addClientDecorators( \Aimeos\Admin\JsonAdm\Iface $clie
 		 * @since 2015.12
 		 * @category Developer
 		 */
-		$decorators = $config->get( 'admin/jsonadm/common/decorators/default', [] );
+		$decorators = $config->get( 'admin/jsonadm/common/decorators/default', ['Standard'] );
 
-		if( $path !== null && is_string( $path ) )
+		if( $path !== '' && is_string( $path ) )
```

Comment:
That may be actual a problem. If you create a PR with:
```
if( $path )
```
only, we are happy to merge it.